### ISSUE TITLE
"File to upload" does not support regex of file name

### ISSUE THREAD

**Issue by fvdcx** [kasunkv/slack-notification#10]
If I input "xx/yy/zz/*.txt", It doesn't work. Only if "xx/yy/zz/abc.txt" can work? Could we support regex?

**Comment by kasunkv** [comment]
@fvdcx Yes, at the moment, only a single file is supported and you need to give the path to that single file. Support for multiple files will be added in a later update.
